scripts/scil_compute_fodf_metrics.py

```diff
@@ -175,13 +139,36 @@ def main():
     assert_inputs_exist(parser, [])
     assert_outputs_exist(parser, args, arglist)
 
-    data, affine = load(args.input)
+    vol = nib.load(args.input)
+    data = vol.get_fdata(dtype=np.float32)
+    affine = vol.affine
+
     if args.mask is None:
-        mask = np.ones(data.shape[:-1])
+        mask = None
     else:
-        mask, affine2 = load(args.mask)
+        mask = np.asanyarray(nib.load(args.mask).dataobj).astype(np.bool)
+        if mask.shape != data.shape[:-1]:
+            raise ValueError("Mask is not the same shape as data.")
+
+    sphere = get_sphere(args.sphere)
 
-    nufo_map, afd_map, afd_sum, peaks_dirs = get_maps(data, mask, args)
+    # Computing peaks
+    peak_dirs, peak_values, \
+        peak_indices = peaks_from_sh(data,
+                                     sphere,
+                                     mask=mask,
+                                     relative_peak_threshold=.5,
+                                     absolute_threshold=args.at,
+                                     min_separation_angle=25,
+                                     normalize_peaks=True,
```

Comment:
If you want afd_max to mean something you must use `normalize_peaks=False` and then normalize them after (before saving)

Right now afd_max = 1 everywhere due to normalisation
PS: please use afd_max rather than afd_map everywhere in the code (afd_max, afd_sum and afd_tot)

Reply:
Working on that